bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/i18n/openwebnet_it.properties

```diff
@@ -236,6 +236,10 @@ channel-type.openwebnet.modeCentralUnit.state.option.WEEKLY = Settimanale
 channel-type.openwebnet.modeCentralUnit.state.option.SCENARIO = Scenari
 channel-type.openwebnet.power.label = Potenza
 channel-type.openwebnet.power.description = Potenza attiva attuale
+channel-type.openwebnet.energyThisMonth.label = Energia Mese
+channel-type.openwebnet.energyThisMonth.decription = Energia del mese
+channel-type.openwebnet.energyToday.label = Energia odierna
+channel-type.openwebnet.energyToday.decription = Energia del giorno
```

Comment:
Please do not update the italian translations. This will be done later in Crowdin.

Reply:
ok I do apologize! I will rollback this